app/models/contrato.rb

```diff
@@ -0,0 +1,44 @@
+require 'httparty'
+
+
+class Contrato < ActiveRecord::Base
+
+
+  def parseContrato
+
+    #Usando a gem HTTParty para realizar a leitura do JSON da API
+    response = HTTParty.get('http://compras.dados.gov.br/contratos/v1/contratos.json')
+      data_hash = response['_embedded']
```

Comment:
Inconsistent indentation detected.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.